docs/using/tutorial/vagrant-setup.rst

```diff
@@ -116,6 +119,18 @@ Note that you will need to make the same substitution in commands used throughou
       Vagrantfile
       alice@mercury:~/flocker-tutorial$
 
+#. Download the cluster and user credentials by right clicking on the link below.
+   Save these to the *flocker-tutorial* directory too, also preserving the file names.
+
+   .. These downloads iare also used in the getting started guide. You will need to adjust
+      it if these downloads are changed.
+   
+   :download:`cluster.crt`
+   
+   :download:`user.crt`
+   
+   :download:`user.key`
```

Comment:
Just above this there is:

```
alice@mercury:~/flocker-tutorial$ ls
Vagrantfile
alice@mercury:~/flocker-tutorial$
```

which shows that you have set things up correctly. That should be moved below here (or duplicated?) with a new `ls`.